login/Models/RoleModel.js

```diff
@@ -0,0 +1,15 @@
+const sq =  require("../DataConnection.js");
+const sequelize = sq.sequelize;
+const Sequelize = sq.Sequelize;
```

Comment:
This two sequelizes are really confusing.

Reply:
You are right,that huge mistake from me